framework/base/Security.php

```diff
@@ -450,6 +446,11 @@ public function generateRandomKey($length = 32)
             throw new InvalidParamException('First parameter ($length) must be greater than 0');
         }
 
+        // always use random_bytes() if it is available
```

Comment:
I moved it behind the checks to allow consistency in exception throwing. PHP7 will cause `\Error` while php5 will cause `yii\base\InvalidParamException`.